jenkins/ci.suse.de/ardana-job.yaml

```diff
@@ -1,11 +1,19 @@
 - job:
     name: ardana-job
     concurrent: true
-    workspace: "$JOB_NAME-$BUILD_ID"
+    workspace: "ardana-job-$BUILD_ID"
     # the selected node needs the cloud credentials
     node: cloud-ardana-ci
     wrappers:
       - timestamps
+      - build-name:
+          name: '#${BUILD_NUMBER}: ${ENV,var="job_name"}/${ENV,var="model"}'
+
+      - timeout:
+          timeout: 60
```

Comment:
Is 60 min enough?

Reply:
happy to raise it when it becomes an issue